Modules/core/Controller/CorespaceaccessController.php

```diff
@@ -118,11 +119,16 @@ public function indexAction($id_space, $letter = "A", $active = "") {
         }
 
         // get user list
+        
         $usersArray = array();
         $isActive = ($active === "active") ? 1 : 0;
         $modelSpaceUser = new CoreSpaceUser();
+        $modelUsersInfo = new UsersInfo();
         $users = $modelSpaceUser->getUsersOfSpaceByLetter($id_space, $letter, $isActive);
         foreach ($users as $user) {
+            $userSupInfos = $modelUsersInfo->get($user['id']);
```

Comment:
not cool here, one request per user....

getUsersIfSpace... should return info via a join (or using an other request with the join)